frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -74,6 +74,31 @@ class ProjectDetails {
     cy.testA11y();
   }
 
+  findDeletedNotebookImage() {
+    return cy.findByTestId('notebook-image');
+  }
+
+  findNotebookImageName() {
+    return cy.findByTestId('image-display-name');
+  }
+
+  findComponent(componentName: string) {
+    return cy.findByTestId(componentName);
+  }
+
+  findEmptyState(componentName: string) {
+    return this.findComponent(componentName).findByTestId('empty-state-title');
+  }
+
+  findDivider() {
```

Comment:
Rename function with `should` prefix and return `this` for chaining. I don't see the return selector as having much value for chaining.